iothub/device/src/ModuleClient.cs

```diff
@@ -577,47 +578,57 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
             InternalClient.SetMessageHandlerAsync(messageHandler, userContext, cancellationToken);
 
         /// <summary>
-        /// Interactively invokes a method on a device
+        /// Interactively invokes a method from an edge module to an edge device.
```

Comment:
@ancaantochi Let me know if this sounds good

Reply:
this, and the 3 API signatures below

Reply:
Confirmed with Anca that invoking methods from a module client only works if you are connected to an edge hub.